scripts/raw_to_easymocap.py

```diff
@@ -0,0 +1,25 @@
+import os, sys, shutil
+
+DATA_TYPE = ['png','PNG','jpg','JPG']
+def create_input():
+    USAGE = "USAGE: python path/to/raw_to_easymocap.py path/to/raw_images path/to/intri.yml path/to/extri.yml"
+    if(len(sys.argv) != 4) or '--help' in sys.argv or '-h' in sys.argv:
+        print(USAGE)
+        os._exit(0)
+    # Raw images path
+    path_img = sys.argv[1]
+    dataset_name = 'easymocap_input'
```

Comment:
why hard-code this one? 

If we really need to hard-coded it, we usually put it on the top of our script right under the import:

```
import os
impors sys

DATASET_NAME = "easymocap_input"
```

Reply:
Added `dataset_name` as input to the script